apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx

```diff
@@ -159,6 +158,7 @@ const ChatRightHandSideLesson = ({
         </ChatButton>
       </span>
       <div ref={endOfDocRef} />
+      <div className="h-[50vh]"></div>
```

Comment:
Adding this enables us to scroll newly inserted elements to the top of the scrolling container